include/pisa/wand_data_compressed.hpp

```diff
@@ -113,27 +116,20 @@ class wand_data_compressed {
                            BlockSize block_size)
         {
             if (seq.docs.size() > configuration::get().threshold_wand_list) {
-                auto t =
-                    block_size.type() == typeid(FixedBlock)
-                        ? static_block_partition(seq,
-                                                 scorer,
-                                                 boost::get<FixedBlock>(block_size).size)
-                        : variable_block_partition(coll,
-                                                   seq,
-                                                   scorer,
-                                                   boost::get<VariableBlock>(block_size).lambda);
+                auto t = block_size.type() == typeid(FixedBlock)
+                             ? static_block_partition(
+                                 seq, scorer, boost::get<FixedBlock>(block_size).size)
+                             : variable_block_partition(
+                                 coll, seq, scorer, boost::get<VariableBlock>(block_size).lambda);
 
                 float max_score = *(std::max_element(t.second.begin(), t.second.end()));
-                for (auto &&s : t.second) {
-                    s /= max_score;
-                }
-                auto ind = compressor_builder.compress_data(t.second);
-
-                compressor_builder.add_posting_list(t.first.size(), t.first.begin(), ind.begin());
-
                 max_term_weight.push_back(max_score);
                 total_elements += seq.docs.size();
                 total_blocks += t.first.size();
+
+                tmp_docs.push_back(std::move(t.first));
```

Comment:
I think `tmp_docs` just a terrible name. I'm a bit confused myself as what it is. Is that the max block documents and `tmp_scores` are the unquantized block max scores? Maybe we should say it then, `unquantized_block_max_scores` and `block_max_documents` or something like that?